src/soter/openssl/soter_ec_key_3.c

```diff
@@ -118,7 +119,7 @@ soter_status_t soter_engine_specific_to_ec_pub_key(const soter_engine_specific_e
     soter_status_t res;
     size_t output_length;
     // 16 is a bit more than necessary for curve identifiers we support
-    char curve_str[16];
+    char curve_str[MAX_CURVE_NAME_LEN];
```

Comment:
Comment talks about 16, but the constant is now 11.

Reply:
Ah, forgot to remove the comment (or put above `#define`)